lib/api.js

```diff
@@ -0,0 +1,123 @@
+const cheerio = require('cheerio');
+
+module.exports = (self, { scanRules }) => {
+  // Defining default values for rules, or getting them from config
+  const MIN_WORDS_NUMBER = (scanRules && scanRules.minWordsNumber) || 350;
```

Comment:
Please use camel case for consistency with other A2 modules.

Reply:
thisIsCamelCase.

Reply:
Done